### PR TITLE
Don't run renovate before we get more minutes

### PR DESCRIPTION
**Pull request by p-j-smith**
- only run renovate the day after we get more minutes

### CODE REVIEW

File: renovate/default-config.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
       "description": "Shorten commit titles",
       "commitMessageTopic": "{{depName}}",
       "matchManagers": ["github-actions", "pre-commit"]
+    },
+    {
+      "description": "Only run the day after we receive more GitHub Actions minutes",
+      "schedule": ["on the seventeenth day of each month"]
```
Comment (p-j-smith) [inline renovate/default-config.json:26]: will this rule be applied to all managers by default?

**Comment by drmatthews** [comment]
I think we need the `schedule` at the same level as `packageRules` rather than setting it as a `packageRule`, which expects to work on a specific dependency (or group).

**Review by drmatthews — APPROVED** [review]
Presumably we have enough action minutes for the workflows that will result from the new tag created after this is merged?

**Comment by p-j-smith** [comment]
Good question. Although I think those workflows should only run next month

**Comment by drmatthews** [comment]
> Although I think those workflows should only run next month

Good point - let's see what happens

**Comment by paddyroddy** [comment]
@p-j-smith this isn't going to work as you expect? At least I don't think so. You've got `    "schedule:monthly"` and `"schedule": ["* * 17 * *"]`. However, presumably CI passed? So it is at least valid config.

Is your claim the second overrides the first? It might be simpler to disable the app, leave `schedule:monthly`, and re-enable after 16th.

**Comment by paddyroddy** [comment]
> Presumably we have enough action minutes for the workflows that will result from the new tag created after this is merged?

So if what @p-j-smith has done works, then no workflows will happen as a result of the tag, right? As for this repo, it is public so not a problem.

**Comment by p-j-smith** [comment]
it turns out the workflows still run, by _prs_ are only opened monthly

**Comment by p-j-smith** [comment]
> Is your claim the second overrides the first?

woops I should have removed that from the `extends`. But yeah I think the `"schedule": ["* * 17 * *"]` will override it. 

>  It might be simpler to disable the app, leave schedule:monthly, and re-enable after 16th.

Yeah I think this is the easiest thing to do for now

**Comment by paddyroddy** [comment]
> run, by _prs_ are only opened monthly

So the branches get made?

**Comment by p-j-smith** [comment]
I think no new branches are made but existing ones are updated

**Comment by paddyroddy** [comment]
> I think no new branches are made but existing ones are updated

Yeah, I agree. So this wouldn't cause problems for us (assuming `renovate` was still on). Because no branches currently exist.

**Pull request merged**